js/main.js

```diff
@@ -30,20 +30,19 @@ var $input = document.querySelector('#searchData');
 
 $input.addEventListener('submit', submitData);
 
-var characterSearched;
-
 function submitData(event) {
   event.preventDefault();
-  characterSearched = {
+  var characterSearched = {
     searched: $input.input.value
   };
+  var name = characterSearched.searched;
```

Comment:
You can just remove this variable entirely, and just pass `$input.input.value` as an argument to `renderChars`.